onboard/nav/rover.cpp

```diff
@@ -160,9 +160,20 @@ bool Rover::turn( double bearing )
 {
     bearing = mod(bearing, 360);
     throughZero( bearing, mRoverStatus.odometry().bearing_deg );
-    if( fabs( bearing - mRoverStatus.odometry().bearing_deg ) < mRoverConfig[ "navThresholds" ][ "turningBearing" ].GetDouble() )
+    if (mRoverStatus.currentState() == NavState::TurnAroundObs ||
+        mRoverStatus.currentState() == NavState::SearchTurnAroundObs ) 
     {
-        return true;
+        if( fabs( bearing - mRoverStatus.odometry().bearing_deg ) < 0 )
+        {
+            return true;
+        }
```

Comment:
We discussed this in person about a week and a half ago but please follow up if needed. Based on our conversation then, I believe we need to add an `else if` for when the bearing is between 0 and some set angle (probably found in the config file so we aren't hardcoding in values) so that we get stuck not giving the rover enough power indefinitely. This `else if` would need to do something to increase the power sent by the joystick LCM. The simple but inelegant solution is to change the bearing to the "some set angle" if in that range.

Reply:
1. I vaguely remember talking about this, but as of now, it will never go into this statement because it fabs. So I think I just didn't make the new config value. 

2. I have no idea how to do the increase in power from the LCM, which I think I told you on that day. It is just changing the upping the turning effort? How much should we do this by?

Reply:
I think I see what you mean, and I will try to put it in the code now

Reply:
Apparently, the ramping problem can be fixed in pid tuning

Reply:
If that's not the error, I honestly have no idea what you're talking about.

Reply:
It would never go into this statement anyway because its the absolute value. So it would never be less than 0